src/content/blog/2023_mitiq_stim_workflow.md

```diff
@@ -0,0 +1,41 @@
+---
+title: ZNE on logical circuits with Mitiq and the Stim backend 
+author: Misty Wahl
+day: 14
+month: 9
+year: 2023
+---
+
+
+In Mitiq v0.29.0 we added a [tutorial demonstrating a method of combining quantum error mitigation (QEM) and quantum error correction (QEC)](https://mitiq.readthedocs.io/en/stable/examples/zne_logical_rb_cirq_stim.html), with the goal of reducing the effective logical error rate of the computation.
+While QEM techniques such as zero noise extrapolation (ZNE) and probabilistic error cancellation are typically thought of as belonging to the NISQ regime, recently it has been shown that they can also benefit applications within the fault-tolerant regime [^1][^2][^3].
+In this example ZNE is applied with noise scaling by global unitary folding on logical randomized benchmarking (RB) circuits, and the use of Mitiq with the Stim stabilizer simulator as the backend is introduced.
+
+## About Mitiq
+[Mitiq](https://mitiq.readthedocs.io/en/latest/) is a compiler that makes programs more robust to errors in quantum computers.
+Because quantum computers have high error rates, compiling for error robustness is critical for useful applications.
+Mitiq is the leading quantum error mitigation compiler and has recently hit 100k downloads and is developed by a community of over 50 contributors worldwide.
+
+## About Stim
+Stim is a quantum stabilizer circuit simulation package, specialized for the simulation of quantum error correction (QEC) circuits.
+It can be used as a Python package, a command line tool, or a C++ library.
+More information can be found on the [Stim project page](https://github.com/quantumlib/Stim). 
+
+Although Mitiq does not support Stim directly as a frontend, it can interface with the Stim backend through the Cirq and Stimcirq packages. Stimcirq is a library for translating between Cirq and Stim.
+
+## A Mitiq + Stim tutorial for applying error mitigation on logical circuits
+
+![Mitiq and Stim logos](/images/mitiq_stim_logo.png)
```

Comment:
Very nice but would be cool to make the image smaller. It's up to you to upload the image filesystem directly or use Cloudinary, but there's a small benefit of Cloudinary over using the filesystem as Cloudinary provides some image optimisation. If you use the filesystem, I recommend reducing a bit the size. You could try embedding some html with size control on the image (or maybe add a bit of white padding to the image?).

Reply:
I didn't see any options to adjust the image on Cloudinary- I guess you meant it does it automatically? I'm using the Cloudinary version now, and both previews look the same to me.

Reply:
Sounds fine to me, @Misty-W!